src/Sylius/Bundle/AttributeBundle/Resources/views/attributeTypes.html.twig

```diff
@@ -8,7 +8,8 @@
             <div class="modal-body">
                 <div class="list-group">
                     {% for name, attributeType in attributeTypes %}
-                        <a href="{{ path('sylius_backend_product_attribute_create', { 'type': name }) }}" id="{{ name }}" class="list-group-item">
+                        {% set route = 'sylius_backend_'~subject~'_attribute_create' %}
```

Comment:
:+1: I think, I'll follow @pjedrzejewski advice and pass ``metadata`` object into these templates.